demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -77,15 +77,16 @@ def __init__(self, pack, pack_path=None, validate_dependencies=False, ignored_er
         self.support = support
 
     # error handling
-    def _add_error(self, error, file_path):
+    def _add_error(self, error, file_path, warning=False):
```

Comment:
did we consider having a `_add_warning` func ?

Reply:
No need, it will look just like the existing function and just utilize the warning flag we have for handle_error. seems redundant to me.